test/uk/gov/hmrc/customsservicestatusfrontend/connectors/CustomsServiceStatusConnectorSpec.scala

```diff
@@ -16,19 +16,20 @@
 
 package uk.gov.hmrc.customsservicestatusfrontend.connectors
 
+import org.mockito.Mockito.*
 import uk.gov.hmrc.customsservicestatusfrontend.helpers.BaseSpec
-import uk.gov.hmrc.customsservicestatusfrontend.helpers.TestData.serviceStatuses
-import uk.gov.hmrc.customsservicestatusfrontend.models.ServiceStatuses
+import uk.gov.hmrc.customsservicestatusfrontend.helpers.TestData.{fakePlannedWork, serviceStatuses}
+import uk.gov.hmrc.customsservicestatusfrontend.models.{OutageData, ServiceStatuses}
 import uk.gov.hmrc.http.client.{HttpClientV2, RequestBuilder}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpReads}
+import uk.gov.hmrc.http.*
 
 import java.net.URL
 import scala.concurrent.{ExecutionContext, Future}
 
 class CustomsServiceStatusConnectorSpec extends BaseSpec {
 
-  val mockHttpClient:    HttpClientV2   = mock[HttpClientV2]
-  val mockRequestHolder: RequestBuilder = mock[RequestBuilder]
+  val mockHttpClient:     HttpClientV2   = mock[HttpClientV2]
+  val mockRequestBuilder: RequestBuilder = mock[RequestBuilder]
```

Comment:
Let's move these into `AllMocks` so that they can be managed from there to align with other services.